internal/plugins/helm/v1/api.go

```diff
@@ -123,7 +123,16 @@ func (p *createAPIPlugin) InjectConfig(c *config.Config) {
 
 // Run will call the plugin actions according to the definitions done in RunOptions interface
 func (p *createAPIPlugin) Run() error {
-	return cmdutil.Run(p)
+	if err := cmdutil.Run(p); err != nil {
+		return err
+	}
+
+	// Run SDK "phase 2" plugins.
+	if err := manifests.RunInit(p.config); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
the `return cmdutil.Run(p)` ensures that we are following the interface. 
The `// Run SDK "phase 2" plugins.` IMO need to be in the PostScaffold actions instead.

Reply:
See https://github.com/operator-framework/operator-sdk/pull/3490#discussion_r459643492.